### PR TITLE
updated the Shared Key Sets section with a working example

### PR DESCRIPTION
**Pull request by smatyas**

### CODE REVIEW

File: symfony-bundle/key-and-key-set-management/key-set-management-jwkset.md [hunk 1]
```diff
@@ -87,15 +87,17 @@ jose:
...
 
 It can be interesting to share your key sets through an Url. This can easily achieved by adding a dedicated controller. This controller is automatically created by the bundle.
 
-You must add the following configuration to your routing file.
+You can enable these routes by adding the following configuration to your routing file.
 
 ```yaml
+# config/routes.yaml
 jwkset_endpoints:
-    resource: "@JoseFrameworkBundle/Resources/config/routing/jwkset_controller.yml"
```
Comment (Spomky) [inline symfony-bundle/key-and-key-set-management/key-set-management-jwkset.md:98]: Indeed the correct path is @JoseFrameworkBundle/Resources/config/routing/jwkset_controller.**php**, not **yml**
I think it is preferrable to keep it. WDYT?

**Review by Spomky — CHANGES_REQUESTED** [review]
Hi,

Thank you for these changes.
I just have one minor comment.

File: symfony-bundle/key-and-key-set-management/key-set-management-jwkset.md (hunk 1 above)
Comment (smatyas) [reply symfony-bundle/key-and-key-set-management/key-set-management-jwkset.md:98]: You're totally right, I missed that file for some reason, but found the routing loader class :)
Now I modified it to work with the **php** resource version.

**Review by Spomky — APPROVED** [review]

**Comment by Spomky** [comment]
Merged.
Many thanks for that PR!

**Pull request merged**